.github/workflows/build_wheels.yml

```diff
@@ -240,6 +240,7 @@ jobs:
       working-directory: apache-beam-source
       env:
         CIBW_BUILD: ${{ matrix.os_python.python }}
+        CIBW_ENVIRONMENT: "SETUPTOOLS_USE_DISTUTILS=stdlib"
```

Comment:
Could you help me understand why we need this (here and below, but especially here since this isn't where its normally failing)?

Reply:
Added it here just as precaution.

Reply:
So the image cibuildwheel uses has a very old setuptools. For the effort to support python 3.10, I would need setuptools>=60 since the stdlib distutils module is deprecated and distutils provided by setuptools is recommended to use. 

Cibuildwheel==1.11.0 uses get-pip.py to install packages. This combined with latest version of setuptools fails because in the setuptools, there is an assertion statement which checks if _distutils is imported or not.

Reply:
Ok, that's good with me - I guess we do the same thing for all our Jenkins jobs - https://github.com/apache/beam/blob/18ab78c8c42a83816a43a3d5252f6d843a82b36c/.test-infra/jenkins/CommonJobProperties.groovy#L100

Would we be better off just setting this as a job level variable? https://docs.github.com/en/actions/learn-github-actions/environment-variables

Reply:
We already set `CIBW_ARCHS_LINUX` like that

Reply:
Hm since this is particular to this block of code and also would be temporary, I think its okay to be here.

Reply:
+1 to a clean up in the master branch (not blocking release). And if any of this is temporary workarounds, please add a link to an issue with a comment.

(In general if a line needs an in depth explanation, or could be removed later (e.g. after python 3.9 deprecation etc.( it would be good to annotate those with a comment.)